public/main.js

```diff
@@ -0,0 +1,99 @@
+
+/***/ }),
+
+/***/ "./source/recipecard/cardExpand.js":
+/*!*****************************************!*\
+  !*** ./source/recipecard/cardExpand.js ***!
+  \*****************************************/
+/***/ ((__unused_webpack_module, __webpack_exports__, __webpack_require__) => {
+
+"use strict";
+eval("__webpack_require__.r(__webpack_exports__);\n/* harmony export */ __webpack_require__.d(__webpack_exports__, {\n/* harmony export */   \"default\": () => (__WEBPACK_DEFAULT_EXPORT__)\n/* harmony export */ });\nclass expandCard {\n    static routes = {};\n    registerCard = (page, func) => {\n        this[page] = func;\n    }\n\n    navigate = (page) => {\n        if (this[page] == undefined) {\n            return console.error(`this page is undefined: ${page}`);\n        }\n        return this[page]();\n    }\n\n    bindCard = (recipeCard, func) => {\n        registerRoute(recipeCard, func);\n        recipeCard.addEventListener('click', () => {\n            Router.navigate(recipeCard);\n        })\n    }\n}\n/* harmony default export */ const __WEBPACK_DEFAULT_EXPORT__ = (expandCard);\n\n//# sourceURL=webpack://cse110-sp21-group11/./source/recipecard/cardExpand.js?");
+
+/***/ })
+
+/******/ 	});
+/************************************************************************/
+/******/ 	// The require scope
+/******/ 	var __webpack_require__ = {};
+/******/ 	
+/************************************************************************/
+/******/ 	/* webpack/runtime/define property getters */
+/******/ 	(() => {
+/******/ 		// define getter functions for harmony exports
+/******/ 		__webpack_require__.d = (exports, definition) => {
+/******/ 			for(var key in definition) {
+/******/ 				if(__webpack_require__.o(definition, key) && !__webpack_require__.o(exports, key)) {
+/******/ 					Object.defineProperty(exports, key, { enumerable: true, get: definition[key] });
+/******/ 				}
+/******/ 			}
+/******/ 		};
+/******/ 	})();
+/******/ 	
+/******/ 	/* webpack/runtime/hasOwnProperty shorthand */
+/******/ 	(() => {
+/******/ 		__webpack_require__.o = (obj, prop) => (Object.prototype.hasOwnProperty.call(obj, prop))
+/******/ 	})();
+/******/ 	
+/******/ 	/* webpack/runtime/make namespace object */
+/******/ 	(() => {
+/******/ 		// define __esModule on exports
+/******/ 		__webpack_require__.r = (exports) => {
+/******/ 			if(typeof Symbol !== 'undefined' && Symbol.toStringTag) {
+/******/ 				Object.defineProperty(exports, Symbol.toStringTag, { value: 'Module' });
+/******/ 			}
+/******/ 			Object.defineProperty(exports, '__esModule', { value: true });
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/cse110-fa21-group11/cse110-sp21-group11/pullRequest?prid=8486342)
```suggestion
/******/ 			Object.defineProperty(exports, "__esModule", { value: true });
```